doc/manual_experimental.rst

```diff
@@ -17,6 +17,11 @@ Some of these are not covered by the ``.experimental`` pragma or
 one may want to use Nim libraries using these features without using them
 oneself.
 
+Whether or not an experimental feature is enabled can be checked in user code
```

Comment:
Ambiguous, you need to clarify (with an example) the difference between:
* whether the compiler supports the flag --experimental:foo
* whether `--experimental:foo` was provided (for which we currently don't have a way to check for, but we should) ; in fact https://github.com/nim-lang/Nim/issues/8644 should be re-opened

Reply:
"Enabling experimental features" doesn't sound that ambiguous to me (I gave an example too, I don't think I have to type "if `--experimental:implicitDeref` is passed or `{.experimental: "implicitDeref".}` is used", and even if I did type that someone is going to go "what about `{.push experimental: "implicitDeref".}`" or whatever), but the confusion is understandable, since it's not obvious how to check for your *first* bullet point while this is trying to document the check for the second bullet point, which you are saying doesn't have a check at all. Maybe this shouldn't be documented if it's such a bad check that it might as well not exist